src/window/macos.rs

```diff
@@ -246,21 +243,14 @@ impl MacosWindowFeature {
                 log::info!("background_color changed to {}", background_color);
                 self.update_background(window, false);
             }
-            #[cfg(target_os = "macos")]
             WindowSettingsChanged::ShowBorder(show_border) => {
                 log::info!("show_border changed to {}", show_border);
                 self.update_background(window, true);
             }
-            #[cfg(target_os = "macos")]
             WindowSettingsChanged::Transparency(transparency) => {
                 log::info!("transparency changed to {}", transparency);
                 self.update_background(window, true);
             }
-            #[cfg(target_os = "macos")]
-            WindowSettingsChanged::WindowBlurred(window_blurred) => {
-                log::info!("window_blurred changed to {}", window_blurred);
-                self.update_background(window, true);
-            }
```

Comment:
I think we still need to keep this to call `update_background` , I seem to remember that was required from the discussions here:
* https://github.com/neovide/neovide/pull/2168

@crupest might know better

Reply:
Right, update_ns_background still uses it, it's just updating the winit setting that's superfluous.

Reply:
Actually this was introduced in #2351 by @emonadeo . I'm not really quite familiar with the codes here.